backend/src/org/akvo/lumen/middleware.clj

```diff
@@ -1,25 +1,26 @@
 (ns org.akvo.lumen.middleware
-  ""
-  (:require
-   [clojure.string :as s]
-   [akvo.commons.jwt :as jwt]
-   [clj-http.client :as client]
-   [ring.util.response :as response]))
+  (:require [akvo.commons.jwt :as jwt]
+            [clj-http.client :as client]
+            [clojure.pprint :refer [pprint]]
```

Comment:
Still the exception doesn't need to be `pprint`ed ? ... perhaps you want (.printStackTrace e) ?